packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/outbound.ts

```diff
@@ -0,0 +1,44 @@
+// The contents of this file were taken from the AWS CDK provider framework.
+// https://github.com/aws/aws-cdk/blob/c52ff08/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/outbound.ts
```

Comment:
Modifications:
- Removed `defaultInvokeFunction` / `invokeFunction`. These are used by the CDK's provider framework to invoke the Lambda functions we defined. The new design doesn't involve Lambda functions invoking other Lambda functions.